universal-application-tool-0.0.1/test/forms/MultiOptionQuestionFormTest.java

```diff
@@ -92,4 +93,39 @@ public void getBuilder_emptyStringMinMax_noPredicateSet() throws Exception {
 
     assertThat(actual).isEqualTo(expected);
   }
+
+  @Test
+  public void getBuilder_setsNextIdCorrectly() throws Exception {
+    MultiOptionQuestionForm form = new DropdownQuestionForm();
+    form.setQuestionName("name");
+    form.setQuestionDescription("description");
+    form.setQuestionText("What is the question text?");
+    form.setQuestionHelpText("help text");
+    form.setMinChoicesRequired("");
+    form.setMaxChoicesAllowed("");
+    form.setOptions(ImmutableList.of("one", "two"));
+    form.setOptionIds(ImmutableList.of(4L, 1L));
+
+    form.getBuilder();
+
+    assertThat(form.getNextAvailableId()).isEqualTo(OptionalLong.of(5L));
+  }
+
+  @Test
+  public void getBuilder_addNewOptions_setsNextIdCorrectly() throws Exception {
+    MultiOptionQuestionForm form = new DropdownQuestionForm();
+    form.setQuestionName("name");
+    form.setQuestionDescription("description");
+    form.setQuestionText("What is the question text?");
+    form.setQuestionHelpText("help text");
+    form.setMinChoicesRequired("");
+    form.setMaxChoicesAllowed("");
+    form.setOptions(ImmutableList.of("one", "two"));
```

Comment:
Suggestion: add some comments as the next line isn't very "readable" without understanding exactly what setOptionsIds is doing, and also that you're skipping a few IDs for some reason.

EG:  // Add two existing options with IDs 4 and 1.

I'd recommend not adding that complexity here since this test is about adding ids, not setting the original next ID correctly, which is already tested in the previous test.

Reply:
I changed the test case so that the IDs are straightforward (1, 2, 3, 4), let me know if that wasn't what you had in mind.